source/llvm/config.d

```diff
@@ -126,7 +133,9 @@ immutable LLVM_AsmParsers = {
 
 /// LLVM Targets with Disassembler capability (if enabled)
 immutable LLVM_Disassemblers = {
-	       static if (LLVM_Version >= asVersion(3, 9, 0)) {
+	static if (LLVM_Version >= asVersion(3, 9, 0)) {
```

Comment:
Oh copy & paste error ... You have already merged this PR, should I open a new one?